src/cares_wrap.cc

```diff
@@ -1216,13 +1216,19 @@ static void CaresTimerClose(Environment* env,
 }
 
 
+static void InitAresOnce() {
+  int r = ares_library_init(ARES_LIB_INIT_ALL);
```

Comment:
I understand this code has been moved as it is, but can we simply use the RHS in the `CHECK_EQ` below?